pkg/ebpf/btf.go

```diff
@@ -240,7 +240,7 @@ func (b *orderedBTFLoader) getEmbeddedBTF(platform, platformVersion, kernelVersi
 	for _, kvp := range kernelVersionPatterns {
 		if kvp.pattern.MatchString(kernelVersion) {
 			// remove possible paths that do not match possible platforms
-			slices.DeleteFunc(possiblePaths, func(s string) bool {
+			possiblePaths := slices.DeleteFunc(possiblePaths, func(s string) bool {
```

Comment:
I suspect a nasty bug here using `:=` instead of `=`.

The new local var `possiblePaths` is shadowing the real `possiblePaths`, and doesn't fail compilation as unused because it's read at line 247.

But the original `possiblePaths` is then left in an inconsistent state.